Mathlib/Algebra/Group/Defs.lean

```diff
@@ -1280,6 +1371,3 @@ initialize_simps_projections Group
 initialize_simps_projections AddGroup
 initialize_simps_projections CommGroup
 initialize_simps_projections AddCommGroup
-
-assert_not_exists Function.Injective
-assert_not_exists IsCommutative
```

Comment:
And then you can restore these

Reply:
Sorry, I don't understand your message. I can't restore these.